test/integration-test.js

```diff
@@ -20,3 +20,18 @@ test('can prebuild a native module for electron', function (t) {
     t.end()
   })
 })
+
+test('can prebuild a native module for node-webkit', function (t) {
+  if (process.versions.modules === '11') {
+    console.log('Skipping test on node 0.10!')
+    return t.end()
+  }
```

Comment:
You can remove this check, since we no longer test on `0.10`. If you feel like it, you can remove the check in the other test as well.